scripts/generate-accounts/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Validator
+
+This script will help us in generating accounts programmaticaly. 
+
+<br />
```

Comment:
Why do we need it?

Reply:
Title description

Reply:
Use [markdown syntax](https://www.markdownguide.org/basic-syntax/#headings).

Reply:
Removed <br />